### PR TITLE
PR: Add Codebase for DappLooker Python SDK

### PR DESCRIPTION
**Pull request by 0xSumitBanik**
## Overview

This pull request introduces the Python Software Development Kit (SDK) for DappLooker under the python directory. The SDK facilitates seamless integration with DappLooker's services and provides developers with an efficient way to access and interact with the platform's features. This initial commit includes the foundational structure and basic functionalities of the Python SDK, laying the groundwork for further enhancements and contributions. Let's build on this and empower the Python community to harness the potential of DappLooker effortlessly. 🚀🐍

### CODE REVIEW

File: python/dapplooker/dapplooker_charts.py [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        chart_uuid (str): The UUID of the chart.
+        api_key (str): The API key for DappLooker.
+        format (str, optional): The format of the response. Defaults to "json".
+
+    Returns:
+        dict: The response from the DappLooker API.
+    """
+
+    @staticmethod
+    def get_chart_data(chart_uuid: str, api_key: str, format: str = None) -> dict:
```
Comment (realchoubey) [inline python/dapplooker/dapplooker_charts.py:19]: Response is list of dict.

**Review by realchoubey — COMMENTED** [review]
- Add new line at the end of all files.
- Add tested chart details and response.

File: python/README.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+3. **Access Blockchain Data**:
+
+   With the SDK imported, create an instance of the `DappLookerChartsAPI` class and make API calls using your API key.
+
+   ```python
+   # Replace 'your_api_key' with the API key you obtained
+   api_key = 'your_api_key'
+   
+   # Replace 'your_chart_uuid' with the UUID of the chart you want to retrieve data for
+   chart_uuid = 'your_chart_uuid'
```
Comment (realchoubey) [inline python/README.md:57]: Add example with `output_format` as `json`, and also add sample response with and without output format.

File: python/.gitignore [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.vscode
+.pytest_cache
+dist/
+build/
+venv/
+dapplooker.egg-info
```
Comment (realchoubey) [inline python/.gitignore:6]: Add new line

File: python/README.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+   chart_uuid = '0be3ba16-3510-400c-9e9a-95b1a2bf8d97'
+   
+   # Make an API call to get chart data
+   response = DappLookerChartsAPI.get_chart_data(chart_uuid, api_key)
+   
+   # Process the response data as per your requirements
+   print(response)
+   """
+      Response: [{'Block Hash': '0xea5498d905a046f89d193f604037edea42752d889a423e39b72cc08bcff1792a', 'Author': '0x6488cfc95016804e23fd6fd370d8cc1a975d4178', 'Block Number': 462671, 'Timestamp': '2023-08-04T12:27:12Z'}, {'Block Hash': '0x4a72ce8fd915a8a392a2e8680d47684a8e4438fc49b13a5d13c35ebf6539ff24', 'Author': '0x8364134c2856e229fe67d2bbc10c7a3575935b80', 'Block Number': 462670, 'Timestamp': '2023-08-04T12:27:00Z'}, {'Block Hash': '0xcdc22e1ba5e15f546913a141f85bb187d17ff47c1d0a666592b226feda3a8054', 'Author': '0x174eed974591bdfa99cf0036f6de03bc08b943fb', 'Block Number': 462669, 'Timestamp': '2023-08-04T12:26:48Z'}, {'Block Hash': '0xee58ef526755158ef4ef7740279a10f5dd4045ffde062f1233b1f0bf409905bb', 'Author': '0xd4d0a0f4d9370b7aeef9660169839ad5a96e8e01', 'Block Number': 462668, 'Timestamp': '2023-08-04T12:26:36Z'} ... ]
+"""
```
Comment (realchoubey) [inline python/README.md:66]: ReadME doesn't look formatted.

**Comment by 0xSumitBanik** [comment]
Addressed all the review comments.
Please re-review @realchoubey

**Comment by 0xSumitBanik** [comment]
Fixed the type object not iterable issue.
Please feel free to review @realchoubey

**Pull request merged**